server/pairing/payload_manager.go

```diff
@@ -9,6 +9,8 @@ import (
 	"os"
 	"path/filepath"
 
+	"github.com/status-im/status-go/params"
+
 	"github.com/status-im/status-go/api"
 
```

Comment:
Hmm, I wonder if there is something wrong with the linter.

The pattern of imports that it normally enforces is:

Group together in alphabetical

- standard library
- 3rd party librarys
- `ethereum/go-ethereum` & `status-im/status-go`

Reply:
> I wonder if there is something wrong with the linter

it must be.
let me check all the files, any other issue for the review? 🙂